packages/sdks/web-js-sdk/src/types.ts

```diff
@@ -29,6 +29,7 @@ export type CoreSdkConfig = Head<Parameters<CreateCoreSdk>>;
 export type WebSdkConfig = CoreSdkConfig & {
   oidcConfig?: OidcConfig;
   getExternalToken?: () => Promise<string>;
+  customStorage?: CustomStorage;
```

Comment:
naming wise, I think that the prefix `custom` is a bit mouthful and redundant, and not provide too much value here (on a broad level, inputs are configuration)

maybe just storage? or even store?
- this term is also consisent with MDN
<img width="1291" height="682" alt="image" src="https://github.com/user-attachments/assets/5ecd9e4f-95f2-4e4e-97ce-074bc0f6e24e" />
- FYI oidc-ts package uses `stateStorage` - https://github.com/authts/oidc-client-ts/blob/c7765577862f9f4ed337f29cd37b448214dc48ca/src/StateStore.ts#L4


I think that storage is the best, for all inputs, and also for type

Reply:
`Storage` is the type of the object
I think `customStorage` gives good indication that it's optional, and it allows to override the default storage